capsules/detector_safety_gear_openvino/README.md

```diff
@@ -1,9 +1,9 @@
 # About the Capsule
 ## Usage
-This capsule is for detecting safety gears (safety vest and safety hat) in the videos.
+This capsule is for detecting safety equipments (safety vest and safety hat).
```

Comment:
```suggestion
This capsule is for detecting safety equipment (safety vest and safety hat).
```

Reply:
Why not plural thought?

Reply:
`equipment` is a collective noun so it never gets pluralized

https://english.stackexchange.com/questions/70828/what-is-the-plural-of-equipment